obsreport/obsreport_processor.go

```diff
@@ -137,118 +137,127 @@ func NewProcessor(cfg ProcessorSettings) *Processor {
 }
 
 // TracesAccepted reports that the trace data was accepted.
-func (por *Processor) TracesAccepted(ctx context.Context, numSpans int) {
+func (por *Processor) TracesAccepted(ctx context.Context, numSpans int) error {
```

Comment:
We will use otel soon, and not sure if the otel will return error. I would wait a bit on this (phase 2 still has >1 month)

Reply:
Okay! Will close this then and re-do it once the change has been done. Thanks for the heads up!